packages/camera/camera_android_camerax/test/android_camera_camerax_test.dart

```diff
@@ -7,70 +7,36 @@ import 'dart:math' show Point;
 
 import 'package:async/async.dart';
 import 'package:camera_android_camerax/camera_android_camerax.dart';
-import 'package:camera_android_camerax/src/analyzer.dart';
-import 'package:camera_android_camerax/src/aspect_ratio_strategy.dart';
-import 'package:camera_android_camerax/src/camera.dart';
-import 'package:camera_android_camerax/src/camera2_camera_control.dart';
-import 'package:camera_android_camerax/src/camera2_camera_info.dart';
-import 'package:camera_android_camerax/src/camera_control.dart';
-import 'package:camera_android_camerax/src/camera_info.dart';
-import 'package:camera_android_camerax/src/camera_metadata.dart';
-import 'package:camera_android_camerax/src/camera_selector.dart';
-import 'package:camera_android_camerax/src/camera_state.dart';
-import 'package:camera_android_camerax/src/camera_state_error.dart';
-import 'package:camera_android_camerax/src/camerax_library.g.dart';
+import 'package:camera_android_camerax/src/camerax_library.dart';
```

Comment:
Nit: this file is extremely large now. Should we split it up?

Reply:
I kept all the same tests that were here originally. I think switching in this PR would create confusing blames. I can follow this PR up though. I have been thinking about better ways to unit test with ProxyApis.